tutorials/Tab_Navigation/README.md

```diff
@@ -0,0 +1,314 @@
+# Tabbing navigation and keyboard focus
+
+## What is covered
+
+In this tutorial, we will show you how to use tabbing navigation between components via keyboard shortcuts `tab` and `shift + tab`.
+
+## Default `Next/Previous` tabbing navigation
+
+By default, `Next/Previous` tabbed navigation moves focus in composition order (in order of appearance), to see how this works, we can use some of the components that are already focusable by default:`TextField`, `OutlinedTextField`, `BasicTextField`, `CircularProgressIndicator`, `LinearProgressIndicator`.
+
+```kotlin
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.WindowState
+import androidx.compose.ui.window.WindowSize
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+fun main() = application {
+    Window(
+        state = WindowState(size = WindowSize(350.dp, 450.dp)),
+        onCloseRequest = ::exitApplication
+    ) {
+        Box(
+            modifier = Modifier.fillMaxSize(),
+            contentAlignment = Alignment.Center
+        ) {
+            Column(
+                modifier = Modifier.padding(50.dp)
+            ) {
+                for (x in 1..5) {
+                    val text = remember { mutableStateOf("") }
+                    OutlinedTextField(
+                        value = text.value,
+                        singleLine = true,
+                        onValueChange = { text.value = it },
+                        modifier = Modifier.height(46.dp)
```

Comment:
Let's not set height explicitly, as on Windows (1.25 scale) we have this:
![image](https://user-images.githubusercontent.com/5963351/127835286-a64e0f64-780b-4e95-aeca-1e07d79823d2.png)

Reply:
Done.